pkgs/data/icons/candy-icons/default.nix

```
@@ -0,0 +1,26 @@
+{ pkgs }:
+
+pkgs.stdenv.mkDerivation {
+
+  pname = "candy-icons";
+  version = "0.6.8";
+
+  nativeBuildInputs = [ pkgs.gtk3 ];
+
+  src = pkgs.fetchurl {
+    url = "https://github.com/EliverLara/candy-icons/archive/37b4566b4bfcdad99131a236ef1fc74db89a302e.zip";
+    sha256 = "fgey2MduS9alUKiFhZr6KTKhYa68zh6LzEIcg/amrOo=";
+  };
+
+  dontUnpack = true;
```

Let's use `fetchzip` to fix this stuff for us automatically. Also, SRI hashes using the `hash` attribute is the norm these days.

```suggestion
  src = fetchzip {
    url = "https://github.com/EliverLara/candy-icons/archive/37b4566b4bfcdad99131a236ef1fc74db89a302e.zip";
    hash = "sha256-fgey2MduS9alUKiFhZr6KTKhYa68zh6LzEIcg/amrOo=";
  };
```